### PR TITLE
WRN-17763: Updated async ui tests for ContextualPopupDecorator

### PR DESCRIPTION
**Pull request by vladut-clapou-lgp**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [X] Documentation was verified or is not changed
* [X] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Async migration of tests for ContextualPopupDecorator

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-17763

### Comments
Enact-DCO-1.0-Signed-off-by: Clapou George-Vladut [vladut.clapou@lgepartner.com](mailto:vladut.clapou@lgepartner.com)

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

File: tests/ui/apps/ContextualPopupDecorator/ContextualPopupDecorator-View.js [hunk 1]
```diff
@@ -10,31 +10,61 @@ const ContextualButton = ContextualPopupDecorator(Button);
...
-					onClick={() => this.clickHandler({open: !open})}
-					onClose={() => this.clickHandler({open: false})}
-					open={open}
-					popupComponent={renderPopup}
+					onClick={() => this.clickHandler({button1Open: !button1Open})}
+					onClose={() => this.clickHandler({button1Open: false})}
+					open={button1Open}
+					popupComponent={renderPopup1}
+				>
+					Contextual Popup Button1
```
Comment (dongsuwon) [inline tests/ui/apps/ContextualPopupDecorator/ContextualPopupDecorator-View.js:47]: I think button name should be changed. It's hard to know what popup button it is.
```suggestion
				default Contextual Popup
```

File: tests/ui/apps/ContextualPopupDecorator/ContextualPopupDecorator-View.js [hunk 2]
```diff
@@ -10,31 +10,61 @@ const ContextualButton = ContextualPopupDecorator(Button);
...
+				<ContextualButton
+					id="button2"
+					onClick={() => this.clickHandler({button2Open: !button2Open})}
+					onClose={() => this.clickHandler({button2Open: false})}
+					open={button2Open}
+					popupComponent={renderPopup2}
 					spotlightRestrict="self-only"
 				>
-					Contextual Popup Button
+					Contextual Popup Button2
```
Comment (dongsuwon) [inline tests/ui/apps/ContextualPopupDecorator/ContextualPopupDecorator-View.js:57]: Same above.
```suggestion
					self-only Contextual Popup
```

File: tests/ui/apps/ContextualPopupDecorator/ContextualPopupDecorator-View.js [hunk 3]
```diff
@@ -10,31 +10,61 @@ const ContextualButton = ContextualPopupDecorator(Button);
...
+				</ContextualButton>
+				<ContextualButton
+					id="button3"
+					noAutoDismiss
+					onClick={() => this.clickHandler({button3Open: !button3Open})}
+					onClose={() => this.clickHandler({button3Open: false})}
+					open={button3Open}
+					popupComponent={renderPopup3}
+				>
+					Contextual Popup Button3
```
Comment (dongsuwon) [inline tests/ui/apps/ContextualPopupDecorator/ContextualPopupDecorator-View.js:67]: Same above.
```suggestion
					noAutoDismiss Contextual Popup
```

**Comment by dongsuwon** [comment]
LGTM

**Pull request merged**